deploy/templates/notificationtemplates/sandbox/idlertriggered/notification.html

```diff
@@ -41,11 +41,12 @@
 >
 
     <p>
-        You are receiving this email because one or more of your workloads in the Developer Sandbox has been running for an extended period and has been automatically idled.
+        Thank you for using the Developer Sandbox! We wanted to let you know that we have scaled down one or more of your workloads.
```

Comment:
I like this friendlier tone much more! I think the previous wording makes it seem like something bad happened and that was maybe part of the problem. Maybe we can make it even more clear that this is an expected outcome with something like 

```suggestion
        Thank you for using the Developer Sandbox! We wanted to let you know that we have scaled down one or more of your workloads. This is completely automatic and no action is required by you.
```

but what you have looks good to me as well

Reply:
I appreciate the suggestion! Let me tweak a little bit